src/setup.erl

```diff
@@ -49,14 +50,20 @@ is_cluster_enabled() ->
         {_,_} -> ok
     end.
 
+is_single_node_enabled(Dbs) ->
```

Comment:
no | ok is an odd contract for something that sounds boolean (why not true | false?)